substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2933,6 +3197,10 @@ mod benches {
 		[pallet_asset_conversion_tx_payment, AssetConversionTxPayment]
 		[pallet_transaction_payment, TransactionPayment]
 		[pallet_election_provider_multi_phase, ElectionProviderMultiPhase]
+		[palllet_election_provider_multi_block, MultiBlock]
```

Comment:
Is the intention to call this pallet `MultiBlock` i.e, not `ElectionProviderMultiBlock`?
I noticed that when I launched the staking-miner...